Apps/Sandcastle/gallery/3D Tiles Feature Styling.html

```diff
@@ -42,45 +42,41 @@
 });
 
 // Load the NYC buildings tileset.
-var tileset = new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(3839) });
+var tileset = new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(4258) });
 viewer.scene.primitives.add(tileset);
 
 // Color buildings based on their height.
 function colorByHeight() {
     tileset.style = new Cesium.Cesium3DTileStyle({
         color: {
             conditions: [
-                ["${height} >= 300", "rgba(45, 0, 75, 0.5)"],
-                ["${height} >= 200", "rgb(102, 71, 151)"],
-                ["${height} >= 100", "rgb(170, 162, 204)"],
-                ["${height} >= 50", "rgb(224, 226, 238)"],
-                ["${height} >= 25", "rgb(252, 230, 200)"],
-                ["${height} >= 10", "rgb(248, 176, 87)"],
-                ["${height} >= 5", "rgb(198, 106, 11)"],
+                ["Number(${height}) >= 300", "rgba(45, 0, 75, 0.5)"],
```

Comment:
We had talked about this earlier, but is the cast required? Does it suggest something is different about the new tileset?